colors/index.js

```diff
@@ -81,6 +82,10 @@ const themeColors = [
     name: 'background',
     value: namedColors.white,
   },
+  {
+    name: 'background-dark',
```

Comment:
it's not really "dark" -- since it's light grey -- but darker than "background", which is white. background-secondary? 

i know we had that convo last week re semantic naming -- definitely an interesting one.